kolibri/core/assets/src/views/sync/SelectAddressModalGroup/api.js

```diff
@@ -27,6 +27,21 @@ function channelIsAvailableAtLocation(channelId, location) {
     });
 }
 
+function versionCompare(version) {
+  //check if version is >= 0.15.0
+  const v = version.split('.');
+  if (parseInt(v[0]) > 0) return true;
+  if (parseInt(v[1]) >= 15) return true;
+  return false;
+}
```

Comment:
may want to consider doing some basic validation of expected version strings and raising an exception if it doesn't match

Reply:
Done